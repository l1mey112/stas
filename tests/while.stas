include <std.stas>

fn main {
	0 1
	while dup 100 <= {
		dup rot + swap
		++
	}
	drop
	putuln
	0 100
	while dup 0 > {
		dup rot + swap
		--
	}
	drop
	putuln
}