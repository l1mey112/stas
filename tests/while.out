5050
5050
